README.rst

```diff
@@ -177,13 +178,7 @@ have been configured and tested:
   - ``profile`` = no coverage testing, but code profiling instead.
     This must be **invoked manually** using the ``-e`` parameter
 
-  **Note 1:** Due to possible 'complications' when setting up PyPy on
-  systems without pyenv, ``pypy3`` tests also will not be automatically
-  run; you must invoke them manually. For example::
-
-    $ tox -e pypy3-nocov
```

Comment:
`pypy3` is now testable on Windows. This greatly simplifies tox invocation in Linux and FreeBSD.